canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -88,7 +89,12 @@ class VirtualizedTable extends React.Component {
 					"index": index,
 					"originalRowIndex": rowData.originalRowIndex,
 					"selected": selected,
-					"isOverSelectOption": this.isOverSelectOption }, evt);
+					"isOverSelectOption": this.isOverSelectOption,
+					"selectMultipleRows": evt.shiftKey ? evt.shiftKey : false,
+					"lastCheckedRow": this.lastChecked === null ? 0 : this.lastChecked }, evt);
+
+				// Track lastChecked row for shift key selection
+				this.lastChecked = rowData.index; // originalRowIndex or index??
```

Comment:
Use originalRowIndex or index??

Reply:
What is the difference?

Reply:
I didn't find any difference between `originalRowIndex` and `index`. I will keep the current code of using `index` for now.